ams/server/src/main/resources/postgres/ams-postgres-init.sql

```diff
@@ -0,0 +1,342 @@
+CREATE TABLE catalog_metadata
+(
+    catalog_id serial PRIMARY KEY,
+    catalog_name varchar(64) NOT NULL,
+    catalog_metastore varchar(64) NOT NULL,
+    storage_configs text,
+    auth_configs text,
+    catalog_properties text,
+    database_count integer NOT NULL DEFAULT 0,
+    table_count integer NOT NULL DEFAULT 0
+);
+CREATE UNIQUE INDEX catalog_name_index ON catalog_metadata (catalog_name);
+
+COMMENT ON TABLE catalog_metadata IS 'Catalog metadata';
+COMMENT ON COLUMN catalog_metadata.catalog_id IS 'Catalog ID';
+COMMENT ON COLUMN catalog_metadata.catalog_name IS 'Catalog name';
+COMMENT ON COLUMN catalog_metadata.catalog_metastore IS 'Catalog type like hms/ams/hadoop/custom';
+COMMENT ON COLUMN catalog_metadata.storage_configs IS 'Base64 code of storage configurations';
+COMMENT ON COLUMN catalog_metadata.auth_configs IS 'Base64 code of authentication configurations';
+COMMENT ON COLUMN catalog_metadata.catalog_properties IS 'Catalog properties';
+
+CREATE TABLE database_metadata
+(
+    catalog_name varchar(64) NOT NULL,
+    db_name varchar(128) NOT NULL,
+    table_count integer NOT NULL DEFAULT 0,
+    PRIMARY KEY (catalog_name, db_name)
+);
+COMMENT ON TABLE database_metadata IS 'Database metadata';
+COMMENT ON COLUMN database_metadata.catalog_name IS 'Catalog name';
+COMMENT ON COLUMN database_metadata.db_name IS 'Database name';
+COMMENT ON COLUMN database_metadata.table_count IS 'Number of tables in the database';
+
+CREATE TABLE optimizer
+(
+    token varchar(50) NOT NULL,
+    resource_id varchar(100) DEFAULT NULL,
+    group_name varchar(50) DEFAULT NULL,
+    container_name varchar(100) DEFAULT NULL,
+    start_time timestamp not null default CURRENT_TIMESTAMP,
+    touch_time timestamp not null default CURRENT_TIMESTAMP,
+    thread_count integer DEFAULT NULL,
+    total_memory bigint DEFAULT NULL,
+    properties text,
+    PRIMARY KEY (token)
+);
+COMMENT ON TABLE optimizer IS 'Optimizer metadata';
+COMMENT ON COLUMN optimizer.resource_id IS 'Optimizer instance ID';
+COMMENT ON COLUMN optimizer.group_name IS 'Group/queue name';
+COMMENT ON COLUMN optimizer.container_name IS 'Container name';
+COMMENT ON COLUMN optimizer.start_time IS 'Optimizer start time';
+COMMENT ON COLUMN optimizer.touch_time IS 'Update time';
+COMMENT ON COLUMN optimizer.thread_count IS 'Total number of all CPU resources';
+COMMENT ON COLUMN optimizer.total_memory IS 'Optimizer memory usage';
+COMMENT ON COLUMN optimizer.properties IS 'optimizer state info, contains like yarn application id and flink job id';
```

Comment:
There is an Index on `group_name` for mysql scripts.

Is the index on "group_name" necessary?